packages/manager/modules/web-universe-components/src/mxplan-banner/mxplan-banner.constants.js

```diff
@@ -0,0 +1,32 @@
+export const GUIDE_LINKS = {
+  CA:
+    'https://help.ovhcloud.com/csm/en-ca-dns-spf-record?id=kb_article_view&sysparm_article=KB0051715',
+  GB:
+    'https://help.ovhcloud.com/csm/en-gb-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061404',
+  DE:
+    'https://help.ovhcloud.com/csm/de-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061403',
+  ES:
+    'https://help.ovhcloud.com/csm/es-es-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061406',
+  FR:
+    'https://help.ovhcloud.com/csm/fr-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061410',
+  IE:
+    'https://help.ovhcloud.com/csm/en-ie-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061405',
+  IT:
+    'https://help.ovhcloud.com/csm/it-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061408',
+  MA:
+    'https://help.ovhcloud.com/csm/fr-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061410',
+  PL:
+    'https://help.ovhcloud.com/csm/pl-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061407',
+  PT:
+    'https://help.ovhcloud.com/csm/pt-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061409',
+  SN:
+    'https://help.ovhcloud.com/csm/fr-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061410',
+  TN:
+    'https://help.ovhcloud.com/csm/fr-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061410',
+  DEFAULT:
+    'https://help.ovhcloud.com/csm/en-gb-dns-spf-record?id=kb_article_view&sysparm_article=KB0051714',
```

Comment:
May be the default page should be this one : https://help.ovhcloud.com/csm/en-gb-mx-plan-zimbra-faq?id=kb_article_view&sysparm_article=KB0061404 ?